Source/GUI/MainWindow.cpp

```diff
@@ -217,9 +225,22 @@ void MainWindow::rerollPredictor()
       SPokemonRNG::getCurrentSystem()->predictStartersForNbrSeconds(
           m_currentSeed, SConfig::getInstance().getPredictionTime());
   m_predictorWidget->setStartersPrediction(predictions, selection);
-  m_predictorWidget->filterUnwanted(m_chkFilterUnwantedPredictions->isChecked());
+  found = m_predictorWidget->filterUnwanted(m_chkFilterUnwantedPredictions->isChecked());
   m_rerollCount++;
   m_lblRerollCount->setText(QString::number(m_rerollCount));
+  return found;
+}
+void MainWindow::autoRerollPredictor()
+{
+  bool found = false;
+  int maxRolls = SConfig::getInstance().getMaxAutoRerolls();
```

Comment:
I disagree on adding a setting for this.  The only reason i would see to limit it is if the number is insanely high (like 10k) because for a number this huge, I would start to have concerns about the speed.  From a user perspective, this is uneeded because there`s the reroll counter and we can always tell the user how many rerolls that did and from that point, they can judge if they take it or not. Even if we have a reroll counter timeout set to 10k, at the very least, this needs to be informed via an error messageBox. 

I just don't think this belongs to a setting (it can be arbirtrary, but VERY high) and I do not think we benefit much from having the user configure it where they can just use their judgement: the button should do its job, but whether or not the user would be okay with it, that`s them to decide after they click on it.